Emby/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "alexbcberio",
+    "id": "202915432175239169"
+  },
+  "url": "emby.media",
+  "description": {
+    "en": "Your personal media on any device. Bringing all of your home videos, music, and photos together into one place has never been easier. Your personal Emby Server automatically converts and streams your media on-the-fly to play on any device."
+  },
+  "service": "Emby",
+  "version": "1.0",
+  "logo": "https://imgur.com/download/mTHBDPm",
+  "thumbnail": "https://imgur.com/download/a80aJAx",
+  "color": "#52B54B",
+  "category": "videos",
+  "tags": [
+      "video",
+      "movie",
+      "tv",
+      "media"
+  ],
+  "regExp": "([a-z0-9]+[.])*([a-z0-9]+)"
```

Comment:
Don't include a regExp that selects everything, best to just remove this.

Reply:
@Alanexei No thats needed for this, if this only actives on a certain if its all good